### DISCUSSION TITLE
I can't figure out what red dots are shown on the map?

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#1400] [Q&A] [answered]
Hey!
I can't figure out what red dots are shown on the map?
What is the purpose?
Thank yo
![reddots](https://user-images.githubusercontent.com/99086477/152697417-c31e6829-cb3f-4d2f-b317-eceac904802b.png)

**Comment by magicbug** [comment, marked as answer]
Should be fairly self explanatory, but the red hot represents a QSO in the Logbook each dot represents a QSO which is displayed in the table below it.

**Reply by MikleBas** [reply]
Hey!
Yes, I understood this, but it is not clear why out of more than 50 connections only these are shown on the map with dots?
Or the algorithm considers that they are far away relative to my QTH and they should be shown, while others are not. Did I understand the display algorithm correctly?
Thanks for your program! Very high quality and professionally made!

**Comment by dg9vh** [comment]
QSOs are only shown, if grid-locators are submitted in the QSO.

**Comment by magicbug** [comment]
Gridsquares are used for the dots if provided else it will just use the DXCC.

**Comment by MikleBas** [comment]
Found a bug
In case of accidental incorrect introduction of Locator
VR21 throws out an error
See screenshot
![bug](https://user-images.githubusercontent.com/99086477/152811876-06885d7c-f3a4-4914-a5c4-dbea1abe0ea9.png)

**Reply by magicbug** [reply]
Already documented as an issue #1385 and its on the todo list to be fixed
